pkg/api/login.go

```diff
@@ -135,7 +144,15 @@ func (hs *HTTPServer) LoginPost(c *models.ReqContext, cmd dtos.LoginCommand) Res
 
 	if redirectTo, _ := url.QueryUnescape(c.GetCookie("redirect_to")); len(redirectTo) > 0 {
 		result["redirectUrl"] = redirectTo
-		c.SetCookie("redirect_to", "", -1, setting.AppSubUrl+"/")
+		cookie := http.Cookie{
+			Name:     "redirect_to",
+			Value:    "",
+			HttpOnly: true,
+			Path:     setting.AppSubUrl + "/",
+			Secure:   setting.CookieSecure,
+			MaxAge:   -1,
+		}
+		http.SetCookie(c.Resp, &cookie)
```

Comment:
Looks like this code is duplicated in two places, can we extract this to a function named getEmptyRedirectCookie()

Reply:
or just getRedirectCookie(value string)